fcm_devices/fcm.py

```diff
@@ -18,9 +19,20 @@ class FCMBackend(object):
     """You can override this class to customise sending of notifications."""
 
     def send_notification(self, device, **kwargs):
-        push_service = FCMNotification(api_key=app_settings.API_KEY)
-        result = push_service.notify_single_device(
-            registration_id=device.token,
+        credentials = service_account.Credentials.from_service_account_info(app_settings.GOOGLE_SERVICE_ACCOUNT_INFO)
+        # PyFCM supports loading from service file directly, OR passing in credentials.
+        # unfortunately we can't pass in the info to be loaded there, so for now
+        # I'm just adding the oauth client to this lib, loading the creds, then passing them in.
+        push_service = FCMNotification(
+            # Annoyingly service_account_file and project_id are positional args, when really PyFCM should
+            # be checking for the existence of credentials first. Project ID can be inferred from credentials.
+            # but for now we'll just pass it as its own setting. TODO create PyFCM issue/PR to fix
+            service_account_file=None,
+            project_id=app_settings.GOOGLE_PROJECT_ID,
+            credentials=credentials
```

Comment:
This is a real pain, but somewhat understandably `PyFCM` wants you to either specific a filepath from which to load the creds, or just provide them already-loaded, so that's what I'm doing.